test/integration/targets/git/tasks/main.yml

```diff
@@ -36,3 +36,6 @@
 - include_tasks: reset-origin.yml
 - include_tasks: ambiguous-ref.yml
 - include_tasks: archive.yml
+- include_tasks: separate-git-dir.yml
+  when:
+    - git_version.stdout is version("1.7.5", '>=')
```

Comment:
I missed that it should've been `git_version.stdout | version_compare("1.7.5", '>=')`

Reply:
I copied that from the line 34